### ISSUE TITLE
Console warnings with latest react version

### ISSUE THREAD

**Issue by shafaq-kazmi** [oliviertassinari/react-swipeable-views#550]
- [x] I have searched the [issues](https://github.com/oliviertassinari/react-swipeable-views/issues) of this repository and believe that this is not a duplicate.

## Expected Behavior

Everything is fine after updating to react latest version
## Current Behavior

There is a warning in the console

```
react-dom.development.js:12487 Warning: componentWillReceiveProps has been renamed, and is not recommended for use. See https://fb.me/react-unsafe-component-lifecycles for details.

* Move data fetching code or side effects to componentDidUpdate.
* If you're updating state whenever props change, refactor your code to use memoization techniques or move it to static getDerivedStateFromProps. Learn more at: https://fb.me/react-derived-state
* Rename componentWillReceiveProps to UNSAFE_componentWillReceiveProps to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: ReactSwipableView
```

## Steps to Reproduce (for bugs)

1. Update react to the latest version

## Your Environment

| Tech                   | Version |
|------------------------|---------|
| react-swipeable-views  |   0.13.3      |
| React                  |     16.10.2    |
| platform               |         |
| etc                    |         |

**Comment by shilangyu** [comment]
Duplicate of #534

**Comment by caleb-harrelson** [comment]
Workaround implemented in #616, release coming soon.

**Comment by Dar0n** [comment]
This is not completed, the mentioned fix was never published in an official release.
